<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1377</width>
    <height>905</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1371</width>
      <height>781</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Todos</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>371</width>
        <height>51</height>
       </rect>
      </property>
      <property name="title">
       <string>Clustering</string>
      </property>
      <widget class="QRadioButton" name="rbTemperatura">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>30</y>
         <width>171</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Temperatura y flujo</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QRadioButton" name="rbConsumo">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>30</y>
         <width>116</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Consumo</string>
       </property>
      </widget>
     </widget>
     <widget class="QComboBox" name="cbx2">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>370</y>
        <width>171</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="KSeparator" name="kseparator_2">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>340</y>
        <width>841</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="lbl1">
      <property name="geometry">
       <rect>
        <x>750</x>
        <y>40</y>
        <width>461</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbl2">
      <property name="geometry">
       <rect>
        <x>750</x>
        <y>370</y>
        <width>461</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QComboBox" name="cbx1">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>40</y>
        <width>171</width>
        <height>27</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>300</height>
       </size>
      </property>
     </widget>
     <widget class="QWidget" name="wgPlotBot" native="true">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>410</y>
        <width>801</width>
        <height>271</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="plotLayoutBot"/>
     </widget>
     <widget class="KSeparator" name="kseparator">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>-20</y>
        <width>101</width>
        <height>731</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QWidget" name="wgDendograma" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>390</y>
        <width>511</width>
        <height>291</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="dendrogramLayout"/>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>60</y>
        <width>311</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Matriz de distancias entre episodios de sueño</string>
      </property>
     </widget>
     <widget class="QWidget" name="wgDistTable" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>90</y>
        <width>511</width>
        <height>291</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="tableLayout"/>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>770</x>
        <y>0</y>
        <width>261</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Comparación entre episodios de sueño</string>
      </property>
     </widget>
     <widget class="QWidget" name="wgPlotUp" native="true">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>80</y>
        <width>801</width>
        <height>251</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="plotLayoutUp"/>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Sueños diurnos</string>
     </attribute>
     <widget class="QComboBox" name="cbx1Siestas">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>40</y>
        <width>171</width>
        <height>27</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>300</height>
       </size>
      </property>
     </widget>
     <widget class="QComboBox" name="cbx2Siestas">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>370</y>
        <width>171</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>770</x>
        <y>0</y>
        <width>381</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Comparación entre episodios de sueño diurnos (siestas)</string>
      </property>
     </widget>
     <widget class="KSeparator" name="kseparator_4">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>-20</y>
        <width>101</width>
        <height>731</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QWidget" name="wgDendograma_2" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>390</y>
        <width>511</width>
        <height>291</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="dendrogramLayoutSiestas"/>
     </widget>
     <widget class="QLabel" name="lbl2Siestas">
      <property name="geometry">
       <rect>
        <x>750</x>
        <y>370</y>
        <width>461</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbl1Siestas">
      <property name="geometry">
       <rect>
        <x>750</x>
        <y>40</y>
        <width>461</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QWidget" name="wgPlotBot_2" native="true">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>410</y>
        <width>801</width>
        <height>271</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="plotLayoutBotSiestas"/>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>371</width>
        <height>51</height>
       </rect>
      </property>
      <property name="title">
       <string>Clustering</string>
      </property>
      <widget class="QRadioButton" name="rbTemperaturaSiestas">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>30</y>
         <width>171</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Temperatura y flujo</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QRadioButton" name="rbConsumoSiestas">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>30</y>
         <width>116</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Consumo</string>
       </property>
      </widget>
     </widget>
     <widget class="KSeparator" name="kseparator_3">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>340</y>
        <width>841</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>60</y>
        <width>311</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Matriz de distancias entre episodios de sueño</string>
      </property>
     </widget>
     <widget class="QWidget" name="wgPlotUp_2" native="true">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>80</y>
        <width>801</width>
        <height>251</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="plotLayoutUpSiestas"/>
     </widget>
     <widget class="QWidget" name="wgDistTable_2" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>90</y>
        <width>511</width>
        <height>291</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="tableLayoutSiestas"/>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Sueños nocturnos</string>
     </attribute>
     <widget class="QComboBox" name="cbx1Suenos">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>40</y>
        <width>171</width>
        <height>27</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>300</height>
       </size>
      </property>
     </widget>
     <widget class="QWidget" name="wgPlotBot_3" native="true">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>410</y>
        <width>801</width>
        <height>271</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="plotLayoutBotSuenos"/>
     </widget>
     <widget class="KSeparator" name="kseparator_5">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>340</y>
        <width>841</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="wgDistTable_3" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>90</y>
        <width>511</width>
        <height>291</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="tableLayoutSuenos"/>
     </widget>
     <widget class="KSeparator" name="kseparator_6">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>-20</y>
        <width>101</width>
        <height>731</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QWidget" name="wgPlotUp_3" native="true">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>80</y>
        <width>801</width>
        <height>251</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="plotLayoutUpSuenos"/>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>60</y>
        <width>311</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Matriz de distancias entre episodios de sueño</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbl1Suenos">
      <property name="geometry">
       <rect>
        <x>750</x>
        <y>40</y>
        <width>461</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>371</width>
        <height>51</height>
       </rect>
      </property>
      <property name="title">
       <string>Clustering</string>
      </property>
      <widget class="QRadioButton" name="rbTemperaturaSuenos">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>30</y>
         <width>171</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Temperatura y flujo</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QRadioButton" name="rbConsumoSuenos">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>30</y>
         <width>116</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Consumo</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="wgDendograma_3" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>390</y>
        <width>511</width>
        <height>291</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="dendrogramLayoutSuenos"/>
     </widget>
     <widget class="QLabel" name="lbl2Suenos">
      <property name="geometry">
       <rect>
        <x>750</x>
        <y>370</y>
        <width>461</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QComboBox" name="cbx2Suenos">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>370</y>
        <width>171</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>770</x>
        <y>0</y>
        <width>361</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Comparación entre episodios de sueño nocturnos</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Consumos</string>
     </attribute>
     <widget class="QWidget" name="widget" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>1321</width>
        <height>241</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="layout_diario"/>
     </widget>
     <widget class="QComboBox" name="cbx_izq">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>300</y>
        <width>141</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_der">
      <property name="geometry">
       <rect>
        <x>860</x>
        <y>296</y>
        <width>471</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_izq">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>296</y>
        <width>471</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="KSeparator" name="kseparator_7">
      <property name="geometry">
       <rect>
        <x>-10</x>
        <y>270</y>
        <width>1361</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="KSeparator" name="kseparator_8">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>280</y>
        <width>20</width>
        <height>491</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QWidget" name="widget_2" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>340</y>
        <width>640</width>
        <height>380</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="layout_dia_izq"/>
     </widget>
     <widget class="QComboBox" name="cbx_der">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>300</y>
        <width>141</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="widget_4" native="true">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>340</y>
        <width>640</width>
        <height>380</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="layout_dia_der"/>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1377</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="actionAbrir"/>
   </widget>
   <addaction name="menuArchivo"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAbrir">
   <property name="text">
    <string>Abrir</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KSeparator</class>
   <extends>QFrame</extends>
   <header>kseparator.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
